### PR TITLE
Initial flask skeleton commit

### PR DESCRIPTION
**Pull request by zachlipton**
This is an uninteresting skeletal flask app for LTC. It's got one endpoint at `/api/echo` that returns the input data unaltered, but it should be easy to plug in endpoints for all our functions. 

My thought is that we should just take the easy way out and have circleci (currently running tests but not deploys) deploy a single environment of this to elastic beanstalk and not overthink it with stage/prod, terraform, etc..., but figuring out whether I'm being too lazy about that can be a tomorrow conversation.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+          # Bring up the container, and touch a file iff it dies cleanly
+          command: docker run ltcapi && touch .docker_exited_cleanly
+          background: true
+      # Give the container some time to come up and hang out
+      - run: sleep 3
+      # Kill it and make sure we see our file
+      - run: docker stop `docker ps | grep ltcapi | awk '{ print $1 }'`
+      - run: "[ -f .docker_exited_cleanly ]"
+      
+#  deploy:
```
Comment (julia326) [inline .circleci/config.yml:52]: may as well get rid of the commented-out stuff

File: environment.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+name: ltc-processing-apire
```
Comment (julia326) [inline environment.yml]: apire?..

**Review by julia326 — APPROVED** [review]
LGTM! thank you this is perfect for what we need. Agreed on no stage/prod/terraform any of that, this is a good case for a fast easy thing

File: .circleci/config.yml (hunk 1 above)
Comment (zachlipton) [reply .circleci/config.yml:52]: I left it in because I was hoping to uncomment it shortly and have it do deploys on the main branch

**Pull request merged**